app/views/messages/_message.html.erb

```diff
@@ -1,9 +1,12 @@
-<%# locals: (message:, conversation:, scroll_into_view: false, request_id: nil) -%>
-<div 
+<%# locals: (message:, scroll_into_view: nil, message_counter: -1) -%>
```

Comment:
I don't get why `message_counter: -1` is needed but I saw you did that elsewhere

Reply:
This is a workaround for the fact that this variable doesn't exist when the partial is rendered without a collection. When you render with a collection this will be overridden with the actual counter index, but without this magic value means the counter == 0 logic won't be fired.